x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -205,20 +191,15 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps & PropsFromRedux
             ...addToCaseActionItems,
             ...statusActionItems,
             ...exceptionActionItems,
-            ...(agentId ? osqueryActionItems : []),
```

Comment:
This agentID is our ''best for now'' solution in the `event list` to verify if Osquery manager can be installed. More context in `isOsqueryAvailable` function ( I added another comment there).